tests/lava/lib/optimization/solvers/generic/test_solver_cpu_backend.py

```diff
@@ -1,32 +1,31 @@
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
 # See: https://spdx.org/licenses/
+import os
 import unittest
 
 import numpy as np
-from lava.lib.optimization.problems.problems import QUBO
+from lava.lib.optimization.problems.problems import QP, QUBO
 from lava.lib.optimization.solvers.generic.solution_finder.process import (
     SolutionFinder,
 )
 from lava.lib.optimization.solvers.generic.solution_reader.process import (
     SolutionReader,
 )
-from lava.lib.optimization.solvers.generic.monitoring_processes \
-    .solution_readout.process import SolutionReadout
-from lava.lib.optimization.solvers.generic.read_gate.process import ReadGate
 from lava.lib.optimization.solvers.generic.solver import (
-    OptimizationSolver, SolverConfig
+    OptimizationSolver,
+    SolverConfig,
 )
 from lava.lib.optimization.utils.generators.mis import MISProblem
 
 
-class TestOptimizationSolver(unittest.TestCase):
+class TestOptimizationSolverQUBO(unittest.TestCase):
     def setUp(self) -> None:
         self.problem = QUBO(
-            np.array([[-5, 2, 4, 0],
-                      [2, -3, 1, 0],
-                      [4, 1, -8, 5],
-                      [0, 0, 5, -6]]))

```

Comment:
If possible leave the matrix style instead of the nested list looking one

Reply:
this is generated by the linting tool. Impractical to manually correct all matrices in text cases by hand.